tests/SeedTests.elm

```diff
@@ -11,14 +11,19 @@ fixedSeed =
     Random.initialSeed 133742
 
 
+{- This is the expected first int output when running this file with a constant seed.
+
+    (\intFromSeed -> fnvHashString fnvInit "Seed test" |> fnvHash intFromSeed)
+
+-}
 expectedNum : Int
 expectedNum =
-    2800615587
+    -3954212174
```

Comment:
I arrived at it experimentally haha...just picked a seed and watched what came out. 😉

Feel free to change the seed if you like!